appengine-standard-java8/translate-pubsub/src/main/java/com/example/appengine/translate_pubsub/PubSubPublish.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.appengine.translate_pubsub;
+
+import com.google.cloud.ServiceOptions;
+import com.google.cloud.pubsub.v1.Publisher;
+import com.google.gson.Gson;
+import com.google.protobuf.ByteString;
+import com.google.pubsub.v1.PubsubMessage;
+import com.google.pubsub.v1.TopicName;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet(name = "Publish with PubSub", value = "/pubsub/publish")
+public class PubSubPublish extends HttpServlet {
+  private Gson gson = new Gson();
+
+  @Override
+  public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+    Publisher publisher = this.publisher;
+    // construct a pubsub message from the payload
+    final String payload = req.getParameter("payload");
+    Message message = new Message(null);
+    message.setData(payload);
+    PubsubMessage pubsubMessage =
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.5-line-wrapping
I think the breaks should be 4, not 8.

Reply:
mvn verify should tell you, however.

Reply:
Done.